BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -729,6 +730,8 @@ extension BlueprintView {
                     UIView.performWithoutAnimation {
                         controller = NativeViewController(node: child)
                         child.layoutAttributes.apply(to: controller.view)
+                        // So the view has a reasonable size during creation/allocation, do this afterwards.
+                        child.viewDescription.applyBeforeLayout(to: controller.view)
```

Comment:
@watt Thoughts on this? This feels a bit better – `init` is all in a `performWithoutAnimation`, and I'd rather the view have at least a sane frame/bounds before we call the apply method here since some views don't generally work well if their frame/bounds has zero size.